docs/developers/get_started/getting_started_riscv_cmake.md

```diff
@@ -17,29 +17,32 @@ executables that can be run on the target platform.
 
 ## Prerequisites
 
-### Install RISC-V Tools
+You'll need a RISC-V LLVM compilation toolchain and a RISC-V enabled QEMU
```

Comment:
How much of this document is different from the new one you're adding to the website?

I haven't given too much through to the `docs/developers/` directory beyond "it should be cleaned up" :P

Going forward, I'd like to see specific tasks like how to run benchmarks, manage the CI, make and test changes, get set up on a specific board, etc. go in this directory and link out to the website for general topics.

Reply:
The old one also includes the dylib codegen path as well as how to build the payload with the RISC-V `v` extension CPU feature. Just like the user-facing Android doc dropped the Vulkan section, I only put the simplest VMVX example in `riscv.md`